utils/download_test_db.py

```diff
@@ -121,31 +121,88 @@ def download_dataset(dataset):
             # Just a newline.
             print()
 
+def download_caffe2_models ():
+    models = ['densenet121',
+              'inception_v1', 
+              'inception_v2',
+              'lenet_mnist',
+              'resnet50',
+              'shufflenet',
+              'squeezenet',
+              'vgg19', 
+              'zfnet512',
+              'bvlc_alexnet',
+              'en2gr',
+              'quant_resnet50']
+    for modelname in models:
+        for filename in ['predict_net.pbtxt', 'predict_net.pb', 'init_net.pb']:
+            os.system ('wget -nc http://fb-glow-assets.s3.amazonaws.com/models/{}/{} -P  {}'.format(modelname, filename, modelname))
```

Comment:
Use `urlretrieve` instead of calling `wget` through `os.system`. Also, there is a subprocess module that is intended to replace `os.system`: https://docs.python.org/2.7/library/subprocess.html#replacing-os-system.

Reply:
Yeah, much safer to use subprocess (*not* in shell mode) instead of os.system for parameterized data, otherwise if someone adds a model like "resnet2; rm -rf /" and we're in trouble.